app/models/miq_queue.rb

```diff
@@ -174,6 +174,12 @@ def self.get(options = {})
     result
   end
 
+  # This are the queue calls related to worker management which
+  # might not be needed once we use kubernetes for worker/pod management
+  def self.put_deprecated(*args)
```

Comment:
Should we add this when we're actually deprecating it?  I guess I'm not sure why we should do this now if it's not really doing anything.